Makefile

```diff
@@ -1,16 +1,17 @@
-CFLAGS=--ansi --pedantic -std=c++0x -O2
-#CFLAGS=--ansi --pedantic -std=c++0x -g 
-INCLUDE= -I.
-all: pls
 
-pls: pls.cpp 
-	g++ $(CFLAGS) $(INCLUDE) pls.cpp utility.cpp -o pls
+.FORCE:
 
-pls_mpreal: pls.cpp 
```

Comment:
Does this still exist as a thing?  I'm not seeing anything in the new code that looks like a mpreal version.  I'm not asserting that's a problem, just asking.